src/main/java/org/example/daos/JobRoleDao.java

```diff
@@ -10,28 +12,66 @@
 import java.util.List;
 
 public class JobRoleDao {
-    public List<JobRole> getJobRoles() throws SQLException {
-        List<JobRole> jobRoles = new ArrayList<>();
+    public List<JobRoleResponse> getJobRoles() throws SQLException {
+        List<JobRoleResponse> jobRoles = new ArrayList<>();
         try (Connection connection = DatabaseConnector.getConnection()) {
             String query = "SELECT jobRoleId, roleName, location,"
                     + " Capability.capabilityName, Band.bandName, "
                     + "closingDate FROM `job-roles`"
                     + " JOIN Capability using(capabilityId)"
                     + " JOIN Band using(bandId)"
-                    + " WHERE closingDate > current_date();";
+                    + " WHERE statusId = 1;";
             PreparedStatement statement = connection.prepareStatement(query);
             ResultSet resultSet = statement.executeQuery();
             while (resultSet.next()) {
-                JobRole jobRole = new JobRole(
+                JobRoleResponse jobRoleResponse = new JobRoleResponse(
                         resultSet.getInt("jobRoleId"),
                         resultSet.getString("roleName"),
                         resultSet.getString("location"),
                         resultSet.getString("capabilityName"),
                         resultSet.getString("bandName"),
                         resultSet.getDate("closingDate"));
-                jobRoles.add(jobRole);
+                jobRoles.add(jobRoleResponse);
             }
         }
         return jobRoles;
     }
+
+    public List<JobRoleDetailed> getJobRoleById(final int jobRoleId)
```

Comment:
A jobRoleId is a unique field so we only ever have to return a single object (not a list).